cqlDB/seed.js

```diff
@@ -0,0 +1,89 @@
+const fs = require('fs');
+const faker = require('faker');
+
+// use mongodb for generating script
+const db = require('./index.js');
+const Product = require('./model.js');
```

Comment:
If you're splitting the load script into another file, I don't think you need to import the db here.

Reply:
that makes sense. i would only need it in my load script, not the dataGen script. i will take it out.